docs/install/README.md

```diff
@@ -16,20 +16,25 @@ kubectl apply -f https://github.com/tektoncd/pipeline/releases/download/v0.7.0/r
 
 ### Clone this repo
 
-Clone the repo and change into the backend-k8s directory:
+Clone the repo and change into the root directory:
 
 ```bash
 git clone $THIS_REPO
```

Comment:
Add the target directory here so that the next `cd` command makes sense even if $THIS_REPO has another name (e.g. forked):

    git clone $THIS_REPO stewardci-core

Reply:
Changed